teams/all.toml

```diff
@@ -42,3 +42,17 @@ excluded-people = [
     "celaus",
     "opeolluwa",
 ]
+
+# Private channel with all team members, so that we can have an easy way of reaching them.
+[[zulip-streams]]
+name = "all/private"
```

Comment:
would be kool if this was named similar to [the private one for leads](https://github.com/rust-lang/team/blob/797965e4abf5cdba42cb4332ce60b1460e0912a1/teams/leads.toml#L17)

so we would have
- `project members (private)`
- `project leads (private)`

with this pr, we would have
- `all/private`
- `project leads (private)`